proto/regen/ecocredit/basket/v1/events.proto

```diff
@@ -17,8 +17,8 @@ message EventCreate {
   string curator = 2;
 }
 
-// EventPut is an event emitted when credits are put into a basket in return
-// for basket tokens.
+// DEPRECATED: EventPut is an event emitted when credits are put into a basket
+// in return for basket tokens.
```

Comment:
This event is not deprecated and only the `credits` field should be marked as deprecated.

```suggestion
// EventPut is an event emitted when credits are put into a basket
// in return for basket tokens.
```

Reply:
I'm unable to make a suggestion for the `credits` field, which is the field we want to deprecate, but I think we should follow the same format as the sdk.

```
  // credits are the credits that were added to the basket.
  // Deprecated: this field has been deprecated and will always return empty.
  repeated BasketCredit credits = 3;
```

Reply:
We can also say `this field will be removed in the next version` and return the credits in the event. I think either way would be ok but we may want to provide backwards compatibility. What do you think?

Reply:
added & put the depreacted proto option as i saw fields do in the SDK

Reply:
> ...but we may want to provide backwards compatibility. What do you think?

I'm thinking it might be better to provide backwards compatibility in the same way we do for basket queries and we may want to do the same for `amount` as well. See https://github.com/regen-network/regen-ledger/pull/1056#discussion_r861065178.